app/assets/stylesheets/_screen.scss

```diff
@@ -623,3 +624,42 @@ div#keycut-help {
     margin-bottom: 5px;
   }
 }
+
+.project-stories {
+  height: 100%;
+  display: inline-block;
+  position: absolute;
+  padding-top: 2px;
+}
+
+.page-header {
+  padding-bottom: 0px;
+
+  .page-header-title {
+    font-weight: 600;
+  }
+}
+
+.simple-alert {
+  .mi {
+    display: inline-block;
+    width: 100%;
+    text-align: center;
+  }
+}
+
+.unstyled-link {
+  text-decoration: none;
+
+  &:hover {
+    text-decoration: none;
+  }
+}
+
+.darkgrey-2 {
+  color: $darkgrey-2;
```

Comment:
Why is that needed? Why not add the color to the class where you expect it to have this color?